src/components/project/enable-encryption.vue

```diff
@@ -343,11 +343,7 @@ export default {
             [
               "Formuláře nakonfigurované pomocí ručně {submission} klíčů budou tyto klíče nadále používat a musí být ručně dešifrovány.",
               "Chcete-li použít automatický proces centrálního šifrování na těchto formulářích, odeberte konfiguraci {base64RsaPublicKey}."
-            ],
-            "Již nebudete mít možnost zobrazit online náhled odeslaných údajů.",
-            "Již se nebudete moci připojit k datům přes OData.",
```

Comment:
I think these two messages were changed in #970. Because they're no longer translated, the messages after them can also no longer be included.